magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -19,7 +19,7 @@ LST:
 
     increase_psf:
         use: true
-        smeared_light_fraction: 0.125
+        fraction: 0.125
```

Comment:
shouldn't this be off now?

Reply:
Yes, it should be OFF for processing all-sky MCs. Since it is now standard to use them, I disabled this method now. Also, I updated the default settings of `increase_nsb` method as same as those used by lstchain for processing Crab-path all-sky MCs.